### DISCUSSION TITLE
How to correctly specify additional directories to be backed up?

### DISCUSSION THREAD

**Discussion by JeCheeseSmith** [nextcloud/all-in-one#5301] [Questions] [answered]
Hi! I'm a little confused about the explanation on how to add additional volumes or directories to be backed up using automatic backups:, it says in the interface: 

> Be sure to individually specify all storage that you want to back up as storage will not be mounted recursively. E.g. providing / as additional backup directory will only back up files and folders that are stored on the root partition and not on the EFI partition or any other. Excluded by the backup will be caches and a few other directories. If you want to back up the root partition you should make sure to stop all services before the backup so it can run correctly. For automating this see [this documentation](https://github.com/nextcloud/all-in-one#how-to-stopstartupdate-containers-or-trigger-the-daily-backup-from-a-script-externally)

Does this mean sub directories also not get recursively included by specifying a "root" directory, or does this only apply for mounted storage/different partitions?

E.g. If i have the following example directory hierarchy:
```
/home/user/compose
/home/user/compose/cheese.txt
/home/user/compose/fruits/
/home/user/compose/fruits/apple.txt
```
Are all the files being backed up or just `cheese.txt` ?

**Comment by szaimen** [comment, marked as answer]
Hi,

> Does this mean sub directories also not get recursively included by specifying a "root" directory, or does this only apply for mounted storage/different partitions?

It only applies to mounted storage/different partitions.
